src/components/messages/messages-container-textarea/MessagesContainerTextarea.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="send-message-container">
+    <el-form>
+      <el-input
+        v-model="message"
+        :rows="2"
+        :disabled="isLoading"
```

Comment:
Wouldn't `isDisabled` make more sense? I can think of reasons to disable a textarea that have nothing to do with loading? For example: if you clicked a submit button, it can make sense to disable it afterwards.

Reply:
`disabled` here is from 'elment-ui' package, we cannot overwrite it 
see here: https://element.eleme.io/#/en-US/component/input#input-attributes

Reply:
I wouldn't change `disabled`, I would change the `isLoaded` property from the new component to `isDisabled` to keep this a bit more consistent and open for interpretation.

Reply:
In that case it could be in another way. Because the purpose here is to disable the button when the state of the component is loading.
We can probably use a computed property `isSendButtonDisabled` which take care of check the loading statement ?